osu.Game/Database/StandardisedScoreMigrationTools.cs

```diff
@@ -264,9 +266,63 @@ public static long ConvertFromLegacyTotalScore(ScoreInfo score, LegacyBeatmapCon
             switch (score.Ruleset.OnlineID)
             {
                 case 0:
+                    if (score.MaxCombo == 0 || score.Accuracy == 0)
+                    {
+                        return (long)Math.Round((
+                            0
+                            + 300000 * Math.Pow(score.Accuracy, 8)
+                            + bonusProportion) * modMultiplier);
+                    }
+
+                    // Assumption :
+                    // - sliders and slider-ticks are uniformly spread arround the beatmap
+                    //      thus we can ignore them without losing much precision (consider a map of hit-circles only !)
+                    // - the Ok/Meh hit results are uniformly spread in the score
+                    //      thus we can simplify and consider each hit result to be score.Accuracy without losing much precision
```

Comment:
What does "consider each hit result to be `score.Accuracy`" mean?

Reply:
If the player has 80% acc for example, we compute the score as if each hit result value was a `0.8*300 = 340` (for judgements with max value of 300)